transport_nantes/topicblog/templates/topicblog/topicblogbase_send_form.html

```diff
@@ -9,6 +9,16 @@
 
 {% block content %}
 
+    <div class="col-10 mx-auto mt-2">
+        <p>
+            Vous envoyez le contenu suivant :
```

Comment:
I think it would be helpful for you to read more about human interface design and its metaphors.  This isn't the sort of thing that blocks PR approval, but it will surely be the subject of a change request at some point soon when someone sees a better way / gets annoyed with the verbosity.

All information doesn't have to come in the form of declarative sentences, and I think if you explore a bit you'll see that most signals between you and your web interface are far, far more subtle.  This box I'm typing in has two tabs, "write" and "preview", neither says "You can click here to see what your text looks like".  Etc.

Wouldn't it have been enough to show the slug, perhaps something like
```
Envoie du slug <a href={% url ...>
```
and the user has seen the web before, sees that there's an underline, understands that it's therefore a link, that it will open something, understands about hover or touch feedback, etc.  Since it opens in a new tab, you could even add a "new tab" superscript (`fas-external-link`, I think) to make that clear.

Attention is a very precious commodity.  Anything that draws attention away from the task at hand and focuses it on administration is usually a negative.

Reply:
Applied this change, true I don't use icons nearly enough.
I'd rather start from overly explicit to less explicit, but I should try to come with a balance !